Sources/KituraWebSocket/WebSocketConnection.swift

```diff
@@ -323,15 +325,16 @@ extension WebSocketConnection: ChannelInboundHandler {
         return true
     }
 }
-
+public var disconnectedCallbackFlag = false
```

Comment:
Please make this `private` and change the name to something like `disconnectedFired`

Reply:
Isn't this a global variable? If yes, I think there are two issues:
- it's not synchronised so this will be a threading issue
- I would assume that there can be more than one `WebSocketConnection` per process so this doesn't feel quite right.

Reply:
@weissi I have made `disconnectedFired` private. So now there won't be synchronization issue as `channelInactive` and `connectionClosed` are getting called on same eventloop.